package.json

```diff
@@ -124,6 +124,7 @@
     "express-static-gzip": "^1.1.3",
     "gnu-terry-pratchett": "0.0.2",
     "helmet": "^3.14.0",
+    "inspectpack": "^4.2.1",
```

Comment:
Shouldn't be a prod dep, but is due to https://github.com/bbc/simorgh/issues/2531 so it's ok here.